src/handlers/browser/launch.ts

```diff
@@ -154,20 +154,30 @@ export const launchBrowser = async (
   })
 
   if (capture.browser) {
-    browserServer.start(browserWindow)
+    try {
+      await Promise.all([
+        new Promise<void>((resolve, reject) => {
+          process.on('spawn', resolve)
+          process.on('error', reject)
+        }),
+        browserServer.start(browserWindow),
+      ])
```

Comment:
I'm attempting to make the startup sequence a bit more reliable:

1. First we wait for the browser process and websocket server to start.
2. Then we inject the extension